build.sh

```diff
@@ -416,6 +417,58 @@ function build_java()
 
 }
 
+################################################################################
+function clean_documentation()
+{
+    # Remove the content of ${doc_folder}/_build
+    rm -rf ${doc_folder}/_build
+}
+
+function build_documentation()
+{
+    clean_documentation
+
+    # Generate HTML
+    echo ""
+    echo -e "${INFO_TAG} Generating HTML documentation"
+    cd ${doc_folder}
+    ${MAKE_EXE} -f Makefile html > /dev/null 2>&1
+    if [ "$?" != 0 ]; then
+        echo -e "${ERROR_TAG} Failure generating HTML documentation"
+        echo -e "${ERROR_TAG} You will need to install:
+            sudo pip install -U sphinx
+            sudo pip install sphinx_rtd_theme"
+        clean_documentation
+        exit -1
+    fi
+    rm -rf ${doc_folder}/doc/html
```

Comment:
this could cause an error if this does not exist. I usually do this:

touch folder
delete folder
mkdir -p folder

Reply:
I do not understand the error. can you explain to me? 
I did not find this approach in the code.